NEMO/apps/kiosk/templates/kiosk/kiosk.html

```diff
@@ -322,6 +342,36 @@ <h3 id="interlock_message" class="text-center"></h3>
 		{
 			ajax_get('{% url 'kiosk_occupancy' %}', {'occupancy': occupancy}, function(response) { $("#occupancy").html(response); });
 		}
+        {# This function modifies the contents of a style sheet after the page has been loaded. #}
+        function set_style(css_text)
+        {
+            let sheet = document.createElement('style');
+            sheet.type = 'text/css';
+            (document.head || document.getElementsByTagName('head')[0]).appendChild(sheet);
+            return (set_style = function(css_text, node)
+            {
+                if(!node || node.parentNode !== sheet)
+                    return sheet.appendChild(document.createTextNode(css_text));
+                node.nodeValue = css_text;
+                return node;
+            })(css_text);
+        }
```

Comment:
this can now be removed
```suggestion
```

Reply:
Fixed by [f10072d](https://github.com/usnistgov/NEMO/pull/178/commits/f10072d3cf4bcdfd12e4a80726136d00d577decf)